kyuubi-relocated-zookeeper-parent/kyuubi-relocated-zookeeper-36/pom.xml

```diff
@@ -38,23 +38,6 @@ under the License.
         <netty.version>4.1.91.Final</netty.version>
     </properties>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.apache.zookeeper</groupId>
-                <artifactId>zookeeper</artifactId>
-                <version>${zookeeper.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.netty</groupId>
```

Comment:
Just be curious, why netty-bom was needed before ?

Reply:
netty is the transitive deps of Zookeeper